lib/tbot/config/config.go

```diff
@@ -154,9 +155,9 @@ type CLIConf struct {
 	// should be written to
 	ConfigureOutput string
 
-	// Proxy is the teleport proxy address. Unlike `AuthServer` this must
+	// ProxyServer is the teleport proxy address. Unlike `AuthServer` this must
 	// explicitly point to a Teleport proxy.
-	Proxy string
+	ProxyServer string
```

Comment:
💅
We are not changing the value/meaning, but can we add an example of what is expected?
Is it `https://proxy.example.com` or `proxy.example.com` or `proxy.example.com:443` ?

Reply:
Yeah - I'll add a short example

Reply:
cb8d89d9c8